framework/harness/test_service.go

```diff
@@ -12,27 +12,11 @@ import (
 	"sync"
 	"time"
 
+	"github.com/launchdarkly/sdk-test-harness/v2/serviceinfo"
+
 	"github.com/launchdarkly/sdk-test-harness/v2/framework"
 )
 
-// TestServiceInfo is status information returned by the test service from the initial status query.
```

Comment:
This has been refactored out into a new single-purpose `serviceinfo` package because I needed to introduce a circular dependency.